tests/docker/test_addon.py

```diff
@@ -99,10 +99,11 @@ def test_addon_map_folder_defaults(
 
 def test_journald_addon_volatile(coresys: CoreSys, addonsdata_system: Dict[str, Data]):
     """Validate volume for journald option, with volatile logs."""
-    docker_addon = get_docker_addon(
-        coresys, addonsdata_system, "journald-addon-config.json"
-    )
-    volumes = docker_addon.volumes
+    with patch("pathlib.Path.exists", return_value=False):
+        docker_addon = get_docker_addon(
+            coresys, addonsdata_system, "journald-addon-config.json"
+        )
+        volumes = docker_addon.volumes
```

Comment:
I don't like changing the test in the same PR like this but I think its necessary in this case. I believe this test was only passing because of my typo. Now that I have used the correct path I suspect the path does actually exist on the test system. That causes this test to fail because its no longer falling back on the volatile path like it was with the bad path.

Patching the external dependency to always return false should ensure this test ensures the functionality we want without the environmental dependency.

Reply:
Makes sense in this case 👍